packages/monitors/test/SyntheticPegMonitor.js

```diff
@@ -14,520 +17,536 @@ const {
   InvalidPriceFeedMock
 } = require("@uma/financial-templates-lib");
 
-contract("SyntheticPegMonitor", function() {
```

Comment:
Large diff in these tests again due to indent on adding additional decimal tests.